gotpl/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang
+
+RUN go get -v github.com/tsg/gotpl
```

Comment:
Does `go get` build a statically compiled binary by default? It seems to work here once you throw it in a scratch container but I don't think this pattern would work for everything. Do we need to call out `export CGO_ENABLED=0`?

Reply:
Go packages are statically compiled by default, with packages directly requiring the relevant bits of cgo functionality being a relatively rare exception. Unfortunately even one not-very-go-tastic package being a dependency taints the whole thing, so it isn't that uncommon to see, but it seems that the particular pattern I'm aiming at here (bundling an uncommon but very simple tool into a portable form within a scratch container) is unlikely to work with many binaries complex enough to have a large number of dependencies anyway, so I don't know that it would be worth messing with here.

Reply:
Works for me. We can leave it for this one